### PR TITLE
Hotfix master - Brace style

### LINKED ISSUE #301: Sass-lint process fails if selector has no space between it and opening brace
The following causes sass-lint to fail and return "Warning: Cannot read property 'type' of undefined Use --force to continue."
```
a{
  background: red;
}
```
But this is fine (space after the a tag)
```
a {
  background: red;
}
```

### PR DESCRIPTION
**Pull request by bgriffith**
With the release of 1.3 there was an issue with the brace style rule failing if there was no proceeding space.

Fixes #301 

DCO 1.1 Signed-off-by: Ben Griffith gt11687@gmail.com

### CODE REVIEW

**Comment by DanPurdy** [comment]
:+1: Looks good to me, we want to release a hotfix?

**Comment by bgriffith** [comment]
1.3.1? Yep definitely. Though happy for it to wait a day or so for any other hot fixes from the recent release.

**Comment by micahgodbolt** [comment]
Confirmed that this, once pulled into master, fixes my previous error on #301

**Comment by DanPurdy** [comment]
We'll release #310 in 1.3.1 too.

File: lib/rules/brace-style.js [hunk 1]
```diff
@@ -64,61 +101,59 @@ module.exports = {
...
-              if (parser.options.style === 'allman' || parser.options.style === 'stroustrup') {
-                result = helpers.addUnique(result, {
-                  'ruleId': parser.rule.name,
-                  'line': previous.start.line,
-                  'column': previous.start.column,
-                  'message': 'Statement should begin on a new line',
-                  'severity': parser.severity
-                });
-              }
+          // if (previous.is('space')) {
```
Comment (DanPurdy) [inline lib/rules/brace-style.js]: Left a comment in here?

Comment (bgriffith) [inline lib/rules/brace-style.js]: Done

**Pull request merged**